css-view-transitions-1/Overview.bs

```diff
@@ -195,8 +478,24 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 	@keyframes -ua-view-transition-fade-in {
 		from { opacity: 0; }
 	}
+
+	html {
+		view-transition-name: root;
+	}
 	```
 
+	<div class=note>
+		<<@keyframes>> with a name beginning `-ua-` are scoped to the user agent style sheet,
+		meaning that when `-ua-view-transition-fade-out` is used in a regular style sheet as an animation name,
+		it does not refer to the above definition.
```

Comment:
The second resolution on the issue is opposite: "The -ua- names *are* usable in author/user stylesheets."

We specifically discussed whether the keyframes in UA stylesheets should be usable in author stylesheets and the answer was yes. This means there can be a collision and the last rule parsed wins. The assumption was that the "-ua-" prefix would avoid the collisions.

Reply:
ohhhhh, I hadn't realised that at all. Thank you for the correction.

Reply:
I removed the note entirely, since it doesn't need clarifying, since it behaves as normal.